README.md

```diff
@@ -1 +1,130 @@
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 My Enumerable <a name="about-project"></a>
+
+In this project, we created a class and a module and used the module inside the class.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://www.ruby-lang.org/en/">Ruby</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- **A method that compares all of the items with a given criteria**
+- **A method that compares any of the items with a given criteria**
+- **A method that filters**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [N/A](https://yourdeployedapplicationlink.com)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+Clone project
+
+```bash
+git clone https://github.com/paulinagonzalezc/myenumerable-ruby
+
+cd myenumerable-ruby
+```
+
+Run the program
+
+```bash
+ruby my_enumerable.rb
+ruby my_list.rb
+```
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Paulina Gonzalez**
+
+- GitHub: [@paulinagonzalezc](https://github.com/paulinagonzalezc)
+- Twitter: [@paulinagzzc](https://twitter.com/paulinagzzc)
+- LinkedIn: [Paulina Gonzalez Cantu](https://linkedin.com/in/paulina-gonzalez-cantu)
+
+👤 **Amanuel Galema**
+
+- GitHub: [@amexabee](https://github.com/amexabee)
+- Twitter: [@i_amanuel](https://twitter.com/i_amanuel)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/amanuel-galema)
+
+👤 **Osama Ashraf**
+
+- GitHub: [Osama Ashraf](https://github.com/osamaashraf6)
+- Twitter: [Osama Ashraf](https://twitter.com/OsamaAshraf578?t=l75KjrhQgK4h-vSPfgk1gA&s=08)
+- LinkedIn: [Osama Ashraf](https://www.linkedin.com/in/osama-salem-2a046b203)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **Solve edge cases**
+- [ ] **More methods**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/paulinagonzalezc/myenumerable-ruby/issues).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please give it a star 😁🌟✨
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+Microverse.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE.md) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
The link to the readme license is not working, hence leads to a 404 error page. Kindly fix it to the your project a more proffessional appeal. :wink: